ACE/tests/Process_Manager_Test.cpp

```diff
@@ -437,7 +437,11 @@ run_main (int argc, ACE_TCHAR *argv[])
                               mgr,
                               1,
                               4);
+#if defined (ACE_HAS_CPP11)
```

Comment:
This change removes the valid tests for ACE_Time_Value-using API. Maybe revert one of these changes?